cmd/main.go

```diff
@@ -45,6 +54,11 @@ func main() {
 
 func SupersimMain(ctx *cli.Context, closeApp context.CancelCauseFunc) (cliapp.Lifecycle, error) {
 	log := oplog.NewLogger(oplog.AppOut(ctx), oplog.ReadCLIConfig(ctx))
+	ok, minAnvilErr := isMinAnvilInstalled(log)
+
+	if !ok || minAnvilErr != nil {
+		return nil, fmt.Errorf("anvil version %s or higher is required.", minAnvilVersion)
```

Comment:
nit: would be nice to also print `please use foundryup to update to the latest version